05_syntax_lab/02.py

```diff
@@ -1,6 +1,12 @@
-"""
-Write a Python program that randomizes 7 numbers
-and prints their sum total.
-If the sum is divisable by 7, also print the word "Boom"
-"""
-
+
+from random import randint
+
+sum = 0
+numbers = []
+for i in range(7):
+	numbers.insert(i,randint(1,100))
+	sum += numbers[i]
+	

```

Comment:
Notice how `sum` is displayed in a different colour than `numbers` and in the same colour as `range`. The reason is `sum` is already a function in Python. See if you can use it to calculate the sum of all the numbers.